galsim/base.py

```diff
@@ -1247,8 +1237,8 @@ class DeVaucouleurs(GSObject):
 
     def __init__(self, half_light_radius=None, scale_radius=None, flux=1., trunc=0.,
                  flux_untruncated=False, gsparams=None):
-        GSObject.__init__(self, _galsim.SBDeVaucouleurs(scale_radius, half_light_radius, flux,
-                                                        trunc, flux_untruncated, gsparams))
+        GSObject.__init__(self, _galsim.SBSersic(4, scale_radius, half_light_radius, flux,
```

Comment:
First the rationalization:

Because there isn't any implementation advantage to having a C++ SBDeVaucouleurs class.  It was already a subclass of SBSersic, which just hard-coded the n=4 and farmed out to SBSersic **all** of the implementation details.  (Unlike SBExponential, which has a number of efficiency improvements over SBSersic(n=1).)  So it seems more transparent to just use SBSersic with n=4 directly so as not to imply that there is something special about SBDeVaucouleurs that is different from SBSersic(n=4).

Now the real reason I got rid of it and came up with the above rationalization to justify it:

Because there were edge cases of the SBDeVaucouleurs class that weren't being covered, and it was easier to just get rid of it (given that it doesn't really serve any useful purpose) than to add the tests that would cover them.